### PR TITLE
[py-plotly] added version 5.2.2 

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-plotly/package.py [hunk 1]
```diff
@@ -12,11 +12,17 @@ class PyPlotly(PythonPackage):
...
 
+    depends_on('python@3.6:', when='@5.2.2:', type=('build', 'run'))
+
     depends_on('py-setuptools', type='build')
-    depends_on('py-decorator@4.0.6:', type=('build', 'run'))
-    depends_on('py-nbformat@4.2.0:', type=('build', 'run'))
-    depends_on('py-requests', type=('build', 'run'))
-    depends_on('py-six', type=('build', 'run'))
-    depends_on('py-pytz', type=('build', 'run'))
+    depends_on('py-requests',   type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-plotly/package.py]: This is no longer required in the latest release

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-plotly/package.py (hunk 1 above)
Comment (qwertos) [reply var/spack/repos/builtin/packages/py-plotly/package.py]: It's not listed in `install_requires` but `import requests` does show up in `setup.py`

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-plotly/package.py]: Are those run when you install the package or only when installing from a git branch? I see they use lazy imports for most of those. Npm is also required when building from git I think.

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-plotly/package.py]: just tested without it and it installed fine. removed.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**